### ISSUE TITLE
CircleIndicatorView -- dependencies, related, dependents

### ISSUE THREAD

**Issue by jbenet** [jbenet/depviz#10]
A DepIndicator is a circle with a count in the center. The count depends on a function, and so does the color.

- [x] prop determines number
- [x] prop determines color

---

### functions for "dependencies" indicator

```js
function dependenciesNumber(item) {
  // blockers = dependencies - (dependencies that are done)
  if (item.links.blockers.length > 0) {
    return item.links.blockers.length
  }
  return dep.links.dependencies.length
}

function dependenciesColor(item) {
  if (item.links.blockers.length > 0) {
    return green
  }
  return red
}
``` 

### functions for "related" indicator

```js
var relatedNumber = (item) -> item.links.related
var relatedColor = (item) -> gray
```

### functions for "dependents" indicator

```js
var dependentsNumber = (item) -> item.links.dependents
var dependentsColor = (item) -> (item.done ? green : red)
``` 

---

The circle indicators here:

![](https://github.com/jbenet/depviz/raw/master/mocks/images/depviz.mocks.010.jpg)

---

depends on #11

**Comment by jbenet** [comment]
Progress:

![2016-11-20 at 4 14 am](https://cloud.githubusercontent.com/assets/138401/20462639/091cd41e-aed8-11e6-8889-a2a6c13e6921.png)

**Comment by wking** [comment]
Will be fixed by #11.  Can we either add a `depends on #11` or merge #11 and close this?

**Comment by jbenet** [comment]
Added `depends on #11`, merged #11, and closed #10.
